bin/repos-lint

```diff
@@ -37,42 +46,60 @@ def check_repo_spec(spec):
         else:
             raise ValueError('unsupported repo spec type: %s' % type(spec))
     except Exception as e:
-        print(e)
+        print_bad(e)
         return False
 
-    print('OK')
+    print_good('OK')
     return True
 
 def validate_string(s):
     # intentionally disallowing subclasses of str since serialized objects
     # should not be in our configuration file.
     if not type(s) is str:
-        raise ValueError('is not a string')
+        raise ValueError('is not a str')
+
+def validate_bool(b):
+    # intentionally disallowing subclasses of bool since serialized objects
+    # should not be in our configuration file.
+    if not type(b) is bool:
+        raise ValueError('is not a bool')
 
 def validate_product_name(pkg):
     if not product_pattern.match(pkg):
         raise ValueError('invalid EUPS product name')
 
 def validate_repo_dict(spec):
-    supported_keys = ['url', 'ref']
+    mandatory_keys = ['url']
+    optional_keys = ['ref', 'lfs']
 
-    # currently we only support the url and ref keys
-    if all(key in spec for key in supported_keys):
+    if 'url' in spec:
         validate_repo_url(spec['url'])
+    if 'ref' in spec:
         validate_string(spec['ref'])
+    if 'lfs' in spec:
+        validate_bool(spec['lfs'])
+        if spec['lfs'] != True:
+            raise ValueError('the only supported value for lfs is true')
```

Comment:
`false` works with `lsst_build` but the linter is intended to steer folks into using the minimal form.